lib/at_cmd_parser/src/at_params.c

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <limits.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+#include <zephyr.h>
+#include <zephyr/types.h>
+#include <at_params.h>
+#include <kernel.h>
+
+#define AT_PARAMS_CALLOC k_calloc
```

Comment:
Please use the OS native abstractions, without wrappers.
Let non-Zephyr code wrap Zephyr's native abstractions instead.